crates/runtime/src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub mod model;
 pub mod modelformat;
 pub mod modelruntime;
 pub mod modelsource;
+pub mod object_store_registry;
```

Comment:
we have different type of naming convention, e.g. "accelerated_table" or "modelruntime". underscore is clearer imo